src/parse_logcfg.c

```diff
@@ -415,7 +415,7 @@ static int cfg_bandmap(const cfg_arg_t arg) {
 }
 
 static int cfg_cwspeed(const cfg_arg_t arg) {
-    int value;
+    int value = 18;
```

Comment:
This assignment makes the false suggestion that the default speed is set here to 18 WPM.
This value is not used, as either `cfg_integer()` can parse the input and overwrites it or parsing fails and then error is returned.

The actual default WPM is set in `cw_utils.c`
```
int speed = 10;
```
No, it's not 10 WPM. It's then mapped using `char speedstr[]` and the final result is 30.

The general issue is that for historical reasons some numeric values are not stored in plain ints.
So the correct fix for such warnings is to refactor the code. I've it on my list.

Reply:
I think the issue is that cppcheck can't evaluate the expression at line 419 where the address of `value` is passed to be set and assumes that it is not initialized.

I was just picking values for fun but I'd also like to not have this come up every time I want to do some code checking.  Perhaps simply initializing the values to 0 would be acceptable?

Reply:
I guess cppcheck is afraid that the value is never assigned to.
Setting to 0 with an appropriate comment would be fine for the moment.